### ISSUE TITLE
Example using Pytorch dataloader with a Coco Detection dataset is needed

### ISSUE THREAD

**Issue by tobymcclean** [nebuly-ai/nebullvm#177]
Unable to find an example that uses a Pytorch dataset and, more importantly, one that uses a Coco Detection dataset.

When running the Yolov8 example, the following error is the output:

> [ERROR] Unexpected exception AssertionError('The dataloader must include label to measure the metric!') happened during tuning.

The error makes sense, but there is no obvious way to use a real Coco Detection dataset. See for example, the OpenVINO Notebook for Yolov8 : [230-yolov8-optimization.ipynb](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-optimization.ipynb)

**Comment by diegofiori** [comment]
Hello @tobymcclean! Thank you for reporting the error! Could you please share with us the code snippet that generates the error?
